massa-protocol-worker/src/handlers/peer_handler/tester.rs

```diff
@@ -303,6 +327,14 @@ impl Tester {
                                     let db = db.clone();
                                     // receive new listener to test
                                     for (addr, _) in listener.1.iter() {
+                                        if peers_in_test.read().unwrap().contains(addr) {
```

Comment:
We use the parking lot RwLock everywhere in the code it would be nice to keep the standard (and they don't use unwrap !)

Reply:
done :)